### DISCUSSION TITLE
最新的发布说明里说支持QQ轻聊版，这是真的吗？我记得……

### DISCUSSION THREAD

**Discussion by mencall** [huiyadanli/RevokeMsgPatcher#641] [General]
……轻聊版早就不让用了吧？
我以前一直使用轻聊版的，虽然后来相对新版QQ缺失些功能（或有些问题），比如新版表情不识别什么的，但这些问题基本处于可接受状态。
但后来腾讯强制不让登录了，折腾了折腾没什么办法，后来也就放弃使用轻聊版了。
现在还有人能使用轻聊版的吗？看说明似乎说是。
我是希望腾讯能把轻聊版更新下并继续使用的，新版QQ好多功能不需要。

**Comment by huiyadanli** [comment]
支持最后几个轻聊版版本吧。几乎没人用的功能，以至于bug存在大半年也没人反馈。

既然轻聊版已经无法使用了，下个版本我会把轻聊版的内容去除。。。
